hack/bump-common-instancetypes.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -ex
+
+source $(dirname "$0")/common.sh
+source $(dirname "$0")/config.sh
+
+function latest_version() {
+    local org="$1"
+    local repo="$2"
+
+    # The API call sorts releases by creation timestamp, so it is enough to request only a few latest ones.
+    curl --fail -s "https://api.github.com/repos/${org}/${repo}/releases" |
+        jq '.[] | select(.prerelease==false) | .tag_name' |
+        tr -d '"' |
+        sort --version-sort |
+        tail -n1
```

Comment:
https://docs.github.com/en/free-pro-team@latest/rest/releases/releases?apiVersion=2022-11-28#get-the-latest-release

Can you switch this over to `https://api.github.com/repos/${org}/${repo}/releases/latest` that doesn't list pre-release as documented above.

Reply:
ACK, updated again